src/libs/notifications/notifications.ts

```diff
@@ -144,6 +160,14 @@ export const handleLocalNotificationOpening = (
     // by `showLocalNotification` but the user did not clicked on it yet
     if (!event.detail.pressAction) return
 
+    const notificationId = event.detail.notification?.id
+
+    if (notificationId === lastLocalNotificationId) {
+      return
```

Comment:
nit: would be nice to print a `log.error()` here so we can track in Sentry if this hypothetical case is real.